python/lsst/obs/base/_instrument.py

```diff
@@ -553,6 +556,19 @@ def makeExposureRecordFromObsInfo(
             sky_angle = obsInfo.boresight_rotation_angle.degree
     if obsInfo.altaz_begin is not None:
         zenith_angle = obsInfo.altaz_begin.zen.degree
+        azimuth = obsInfo.altaz_begin.az.degree
+
+    extras: Dict[str, Any] = {}
```

Comment:
Reminder that we can now write `dict[str, Any]` with `__future__.annotations` in 3.8, and without in 3.10.

Reply:
Will take some getting used to.